test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -0,0 +1,231 @@
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import static java.awt.print.PageFormat.LANDSCAPE;
+import static java.awt.print.PageFormat.PORTRAIT;
+import static java.awt.print.PageFormat.REVERSE_LANDSCAPE;
+import java.awt.print.Paper;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.Sides;
+import javax.swing.JFrame;
+import jtreg.SkippedException;
+
+/*
+ * @test
+ * @bug 8307246
+ * @key printer
+ * @library ../../../regtesthelpers
+ * @library /test/lib
+ * @build PassFailJFrame
+ * @build jtreg.SkippedException
+ * @summary Test for comparing offsets of images drawn with opaque and translucent colors printed in all orientations
+ * @run main/manual AlphaPrintingOffsets testAlpha
+ */
+
+public class AlphaPrintingOffsets {
+    private static final String INSTRUCTIONS =
+                    "Tested bug occurs only on-paper printing so you mustn't use PDF printer\n" +
+                    "1.Java print dialog should appear.\n" +
+                    "2. Press the Print button on the Java Print dialog.\n" +
+                    "3. Please check the page margin rectangle are properly drawn and visible on all sides on all pages.\n" +
+                    "If so, press PASS, else press FAIL.";
+
+    private static boolean isAlphaTestModeSet = false;
+
+    public static boolean getAlphaTestModeSet() {
+        return isAlphaTestModeSet;
+    }
+
+    public static void main(String[] args) throws Exception {
+        if (PrinterJob.lookupPrintServices().length > 0) {
+
+            String instructionsHeader = "This test prints 6 pages with same image except text messages. \n";
```

Comment:
I am not sure what "except text messages" conveys in the instructions? 
I guess this should be "This test prints 6 pages with page margin rectangle and a text message"
We told the checking criteria later "Please check the page margin rectangle are properly drawn and visible on all sides on all pages"